Circle CI

pat-s opened the pull request.

pat-s committed:
try circle ci

pat-s committed:
push

pat-s committed:
push

pat-s committed:
push

pat-s committed:
push

pat-s committed:
update circleci

pat-s committed:
deploy

pat-s committed:
deploy

pat-s committed:
deploy

pat-s committed:
add cron job

pat-s committed:
try setting env var id_rsa

pat-s committed:
try renv tic

pat-s committed:
Merge branch 'circle-ci' of github.com:pat-s/tic.package into circle-ci

pat-s committed:
test renv

pat-s committed:
use latest tic

pat-s committed:
test deployment

pat-s committed:
do not deploy

krlmlr commented on tic.R:
```diff
@@ -1,5 +1,4 @@
-if (ci_on_travis()) {
-  do_pkgdown()
-}
+do_pkgdown()
```
Are we now building pkgdown on all platforms?

krlmlr reviewed (approved):
Can we get a CircleCI badge in the README?

krlmlr commented on .travis.yml:
```diff
@@ -7,7 +7,7 @@
   - R -q -e 'if (!requireNamespace("remotes")) install.packages("remotes")'
   - R -q -e 'if (getRversion() < "3.2" && !requireNamespace("curl")) install.packages("curl")'
-  - R -q -e 'remotes::install_github("ropenscilabs/tic", upgrade = "always"); print(tic::dsl_load()); tic::prepare_all_stages()'
+  - R -q -e 'remotes::install_github("ropenscilabs/tic@renv", upgrade = "always"); print(tic::dsl_load()); tic::prepare_all_stages()'
```
Do we need this?

pat-s committed:
clean tic.R

pat-s committed:
pkgdown only on travis

pat-s committed:
remove deployment step

pat-s committed:
add badge

pat-s commented:
I cleaned up and also removed the "deployment" stage from circle -> deployment only on Travis for this repo.

The skeleton including deployment will live in the template in `tic.R`.

pat-s committed:
upd circle yaml
